conda-lock.yml

```diff
@@ -31,7 +31,7 @@ package:
   manager: conda
   platform: win-64
   dependencies: {}
-  url: https://conda.anaconda.org/conda-forge/win-64/_libavif_api-1.1.1-h57928b3_0.conda
```

Comment:
I don't know why this has happened. I ran `conda-lock --update matplotlib` to test updating a lockfile..